dali/dfu/dfuwu.cpp

```diff
@@ -795,10 +795,10 @@ class CDFUprogress: public CLinkedDFUWUchild, implements IDFUprogress
         CriticalBlock block(parent->crit);
         queryRoot()->setProp("@subdone",str);
     }
-    void setFileAccessCost(double fileAccessCost)
+    void setFileAccessCost(cost_type fileAccessCost)
     {
         CriticalBlock block(parent->crit);
-        queryRoot()->setPropReal("@fileAccessCost", fileAccessCost);
+        queryRoot()->setPropInt64("@fileAccessCost", fileAccessCost);
```

Comment:
Relaed to comment in ws_fsService.cpp.
The corresponding get function in this class is (unchanged):
```
    double getFileAccessCost() const
    {
        CriticalBlock block(parent->crit);
        return queryRoot()->getPropInt64("@fileAccessCost");
    }
```

that looks like a pre-existing bug. It was set as a real, but retrieved as an int (then cast to a double on return).